iceoryx_posh/include/iceoryx_posh/runtime/service_discovery.hpp

```diff
@@ -44,24 +44,11 @@ class ServiceDiscovery
     findService(const cxx::variant<Wildcard_t, capro::IdString_t> service,
```

Comment:
`findService` could be a static method at the moment, but probably not anymore after the refactoring for this issue is finished. Therefore, I'd like to keep it as it is.